ui/panels-plugin/src/plugins/time-series-chart/YAxisOptionsEditor.tsx

```diff
@@ -51,6 +51,7 @@ export function YAxisOptionsEditor({ value, onChange }: YAxisOptionsEditorProps)
         control={
           <TextField
             value={value.label ?? ''}
+            inputProps={{ 'aria-label': 'enter y axis label' }}
```

Comment:
should we have make these aria-labels as constants so we can use them in e2e tests as well?

Reply:
Yeah definitely makes sense, @juliepagano if I did make this a constant and use it in both places, where do you think that constant should live? My gut says `@perses-dev/core` but is that okay to import from @perses-dev/core in the `e2e` folder?

Reply:
I'm also thinking now that maybe I should move YAxisOptionsEditor.tsx to @perses-dev/components and all the types it uses (DEFAULT_UNIT, DEFAULT_Y_AXIS, YAxisOptions, Y_AXIS_CONFIG) to @perses-dev/core so that we can use it for other panel types like BarChart in the future (although probably best to do that in a separate PR)

Reply:
I'm going to go ahead and merge this and do any type reorganization in a follow up PR!